frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -80,15 +82,18 @@ export const getNodeInfo = (nodes: NodeKind[]) =>
       const memoryRaw = getNodeAllocatableMemory(node);
       const memory = convertToBaseValue(memoryRaw);
       const zone = getZone(node);
+      const rack = getRack(node);
       data.cpu += cpus;
       data.memory += memory;
-      if (zone && hasNoTaints(node)) data.zones.add(zone);
+      if (zone && (hasOCSTaint(node) || hasNoTaints(node))) data.zones.add(zone);
+      if (rack && (hasOCSTaint(node) || hasNoTaints(node))) data.racks.add(rack);
       return data;
     },
     {
       cpu: 0,
       memory: 0,
-      zones: new Set(),
+      zones: new Set<string>(),
+      racks: new Set<string>(),
```

Comment:
racks only required for add capacity , we should not add that to generic utility.